examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -2,22 +2,31 @@
 // in order to be processed by the checkout flow for the "Shipping" integration
 
 global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
-
+    // To access the service below, you may need to add https://b2b-commerce-test.herokuapp.com in Setup | Security | Remote site settings.
+    private static String httpHost = 'https://b2b-commerce-test.herokuapp.com';
```

Comment:
This url should not be a working url and instead should be updated to

http://example.com/path